code/src/index.css

```diff
@@ -5,9 +5,169 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+  color: #ffffff;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* For the whole music albums */
+.music-container {
+  display: grid;
+  grid-template-columns: 100%;
+  margin: auto 0;
+  padding: 2%;
+}
+
+.theAlbums {
+  margin: 20px auto;
```

Comment:
Nice job with centering the albums. This margin: 20px auto; works magic to center the albums on mobile, I will steal this for our project 🙂